datadog/resource_datadog_synthetics_test_.go

```diff
@@ -111,7 +111,44 @@ func syntheticsTestRequest() *schema.Schema {
 
 func syntheticsTestOptions() *schema.Schema {
 	return &schema.Schema{
-		Type:     schema.TypeMap,
+		Type: schema.TypeMap,
+		DiffSuppressFunc: func(key, old, new string, d *schema.ResourceData) bool {
+			if key == "options.follow_redirects" {
+				// TF nested schemas is limited to string values only
+				// follow_redirects being a boolean in Datadog json api
+				// we need a sane way to convert from boolean to string
+				// and from string to boolean
+				truthyValues := map[string]struct{}{"true": {}, "1": {}}
+				falsyValues := map[string]struct{}{"false": {}, "0": {}}
+				_, isTruthyOld := truthyValues[old]
+				_, isFalsyOld := falsyValues[old]
+				_, isTruthyNew := truthyValues[new]
+				_, isFalsyNew := falsyValues[new]
+				if !isTruthyOld && !isFalsyOld {
```

Comment:
Is this just for determining a default value? Like if the old value wasn't a recognized True or False, default to False?

Reply:
Could we refactor this to use the parseBool function - https://golang.org/pkg/strconv/#ParseBool
If there is an error we could default to the value being false. Let me know what you think.